packages/tinybased/src/lib/tinybased.spec.ts

```diff
@@ -21,25 +21,33 @@ const exampleNote = {
 };
 
 const baseBuilder = new SchemaBuilder().addTable(usersTable);
+const twoValueBuilder = new SchemaBuilder()
+  .addValue('online', 'boolean')
+  .addValue('userId', 'string');
 
 describe('tinybased', () => {
   describe('Key-Value API', () => {
-    it('Can be defined', async () => {
-      const builder = new SchemaBuilder()
-        .addValue('online', 'boolean')
-        .addValue('userId', 'string');
-
-      const built = await builder.build();
+    let twoValueBased!: Awaited<ReturnType<(typeof twoValueBuilder)['build']>>;
```

Comment:
Can we use our simplified type here? I think its called `InferTinyBasedFromSchemaBuidler` or something similar to that